src/Assertion/AssertionPluginManager.php

```diff
@@ -34,12 +34,11 @@ class AssertionPluginManager extends AbstractPluginManager
     /**
      * {@inheritDoc}
      */
-    public function validatePlugin($plugin)
+    public function validate($plugin)
     {
         if ($plugin instanceof AssertionInterface) {
             return; // we're okay
         }
-
         throw new Exception\RuntimeException(sprintf(
```

Comment:
maybe not so important, but this is against interface https://github.com/zendframework/zend-servicemanager/blob/master/src/PluginManagerInterface.php#L27-L28
`Zend\ServiceManager\Exception\InvalidServiceException` should be thrown